addons/website_slides/static/src/js/activity.js

```diff
@@ -42,6 +41,5 @@ function applyInclude(Activity) {
 }
 
 applyInclude(KanbanActivity);
-applyInclude(MailActivity);
```

Comment:
Why change this if we didn't convert activity?

Reply:
We have converted activity in chatter, and this `MailActivity` was about activity in chatter.